src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp

```diff
@@ -80,7 +80,6 @@ HydrofractureSolver< POROMECHANICS_SOLVER >::HydrofractureSolver( const string &
   // This may need to be different depending on whether poroelasticity is on or not.
   m_linearSolverParameters.get().mgr.strategy = LinearSolverParameters::MGR::StrategyType::hydrofracture;
   m_linearSolverParameters.get().mgr.separateComponents = false;
```

Comment:
@victorapm how about this one?

Reply:
I need to check this one with @rrsettgast and @CusiniM, but I guess SDC for the fracture displacements is not very useful so I left it false. I can be wrong though

Reply:
we should test it but I think that we should probably keep it on.

Reply:
After discussing about this last week, we decided it should be on. I will update